src/engraving/layout/layoutsystem.cpp

```diff
@@ -920,7 +938,7 @@ void LayoutSystem::layoutSystemElements(const LayoutOptions& options, LayoutCont
                 }
                 layoutTies(c, system, stick);
             }
-        }
+        }*/
```

Comment:
Why not just deleting these lines and let git keep the history?

Reply:
good catch, missed that one